src/components/Copybox.vue

```diff
@@ -39,7 +39,7 @@
 <style scoped>
   .copybox {
     background: #fff;
-    box-shadow: 0px 11px 21px 0px rgba(0,0,0,0.30);
+    box-shadow: 0px 11px 31px 0px #A190AE30;
```

Comment:
General better to use RGBA when using Alphas.

`rgba(161, 144, 174, 0.19)`

Not need to change, but just something to note.

Reply:
fixed